packages/wrangler/src/index.tsx

```diff
@@ -2186,77 +2199,81 @@ export async function main(argv: string[]): Promise<void> {
     }
   );
 
-  wrangler.command("pages", "⚡️ Configure Cloudflare Pages", pages);
+  wrangler.command("pages", "⚡️ Configure Cloudflare Pages", (pagesYargs) =>
+    pages({ pagesYargs, subHelp } as unknown as Argv<unknown>)
```

Comment:
This is a bit magical. Can we just do something like:

```ts
wrangler.command("pages", "...", (pagesYargs) => {
  return pages(pargesYargs).command(subHelp);
}
```

Reply:
I'll give that a try 👍

Reply:
Was able to get it working with this`pages(pagesYargs.command(subHelp))` 😄